code/modules/xenoarcheaology/anomaly_container.dm

```diff
@@ -37,7 +46,16 @@
 	underlays.Cut()
 	desc = initial(desc)
 
+/atom/MouseDrop(var/obj/structure/anomaly_container/over_object)
+	. = ..()
+
+	if(src && istype(loc, /turf) && is_anomalous() && Adjacent(over_object) && CanMouseDrop(over_object, usr))
+		Bumped(usr)
+		over_object.contain(src)
+
+/*
 /obj/machinery/artifact/MouseDrop(var/obj/structure/anomaly_container/over_object)
```

Comment:
Seems a bit unnecessary to keep this commented code here given that the above mousedrop makes it completely redundant.